front/app/containers/Admin/projects/project/analysis/Insights/SummarizeButton.tsx

```diff
@@ -49,13 +47,6 @@ const SummarizeButton = () => {
     );
   };
 
-  const { data: inputs } = useInfiniteAnalysisInputs({
-    analysisId,
-    queryParams: filters,
-  });
-
-  const inputsCount = inputs?.pages[0].meta.filtered_count || 0;
```

Comment:
This is the same as filteredInputsCount from the parent so used that instead. Makes it easier to follow the data flow.